README.md

```diff
@@ -1,47 +1,68 @@
-Mojio
+MojioEvents
 ===========
-
-Get information about your [moj.io](http://moj.io) connected vehicles. There are two blocks:
- * [MojioVehicles](#mojiovehicles)
- * [MojioEvents](#mojioevents)
+Retrieve each new event from Mojio.
 
 Properties
---------------
- * **API Key** - This is your API token returned by Moj.io. See the [Getting API Access](#getting-api-access) section for details
+----------
+- **creds**: Credentials to connect to Mojio API.
+- **include_query**: Whether to include queries in request to Mojio.
+- **polling_interval**: How often Mojio is polled. When using more than one query. Each query will be polled at a period equal to the *polling interval* times the number of queries.
+- **queries**: Inherited from base rest polling block but not used.
+- **retry_interval**: When a url request fails, how long to wait before attempting to try again.
+- **retry_limit**: Number if times to retry on a poll.
 
-Dependencies
-----------------
-None
+Inputs
+------
+- **default**: Any list of signals.
+
+Outputs
+-------
+- **default**: One signal for each poll for each **new** event since the last poll. The schema for the vehicle response can be found through the [Moj.io Schema API Call](https://api.moj.io/v1/Schema?entityType=Event).
 
 Commands
-----------------
+--------
 None
 
-Input
--------
-Input signals will trigger REST polls.
-
 Getting API Access
----------
+------------------
```

Comment:
`Getting API Access` section is duplicated after each block.  Probably only needs to be at bottom of README

Reply:
We don't really have a way to do this though...

Reply:
At least with `nio buildreadme`